packages/client-sdk-nodejs/package.json

```diff
@@ -23,8 +23,8 @@
     "integration-test-storage": "jest storage/ --maxWorkers 1 -- useConsistentReads",
     "integration-test-store": "npm run integration-test-storage",
     "integration-test-topics": "jest topics/ webhooks/ --maxWorkers 1 -- useConsistentReads",
-    "integration-test-retry": "jest retry/automated-retry-tests.test.ts --maxWorkers 1 -- useConsistentReads",
-    "integration-test-consistent-reads": "jest integration --testPathIgnorePatterns=\"automated-retry-tests\\.test\\.ts\" --maxWorkers 1 -- useConsistentReads",
+    "integration-test-retry": "jest retry/ --maxWorkers 1 -- useConsistentReads",
+    "integration-test-consistent-reads": "jest integration --testPathIgnorePatterns=\"retry/\" --maxWorkers 1 -- useConsistentReads",
```

Comment:
I split the retry integration tests into multiple files, one for each retry strategy.